src/Aspire.Dashboard/appsettings.Development.json

```diff
@@ -4,5 +4,6 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
-  }
+  },
+  "DetailedErrors": true
```

Comment:
?

Reply:
This will disappear, this change was made on main recently and its useful for debugging issues.

Reply:
@davidfowl it seems like this is a helpful setting to have in the development appsettings. It causes the browser to show more detailed exception messages when the Blazor circuit fails. Any concern here?

Reply:
Seems fine as long as we don't run the dashboard development outside of the repo.